samples/openthread/coap_client/CMakeLists.txt

```diff
@@ -10,6 +10,8 @@ set(NRF_SUPPORTED_BOARDS
 	nrf52833dk_nrf52833
 )
 
+set(CONF_FILE "prj.conf ../openthread.conf")
```

Comment:
This is not acceptable, users should always be able to specify `-DCONF_FILE=<somefile.conf>` on command line when invoking CMake (or `west build`)